src/hive_mind_bridge/include/hive_mind_bridge/MessageUtils.h

```diff
@@ -90,6 +90,14 @@ namespace MessageUtils {
                                          UserCallTargetDTO moduleDestination,
                                          std::string callbackName);
 
+    /**
+     * Create a greet message
+     * @param msgSourceId The swarm id of this robot. If the ID is not set yet, the default value 0
+     * should be sent.
+     * @return The created message
+     */
+    MessageDTO createGreetMessage(uint32_t msgSourceId = 0);
```

Comment:
Shouldn't pass a destination too? even though it would be with a default value of 0 and probably not used. If not it should be name createLocalGreetMessage I think

Reply:
Actually, there is no need to pass any value here, since every ID will be set to 0. I changed this a bit in the last commit.